composer.json

```diff
@@ -10,22 +10,28 @@
   ],
   "require": {
     "php": "^8.1",
-    "symfony/http-kernel": "^6.2",
-    "illuminate/support": "^10.0",
-    "illuminate/database": "^10.0",
-    "illuminate/validation": "^10.0",
+    "symfony/http-kernel": "^6.2|^7.0",
+    "illuminate/support": "^10.0|^11.0",
+    "illuminate/database": "^10.0|^11.0",
+    "illuminate/validation": "^10.0|^11.0",
     "league/mime-type-detection": "^1.9"
   },
   "require-dev": {
-    "psy/psysh": "@stable",
+    "psy/psysh": "^0.11.22|^0.12",
     "mockery/mockery": "^1.3.1",
-    "phpunit/phpunit": "^9.0",
-    "laravel/framework": "^10.0",
-    "orchestra/testbench": "^8.0",
-    "orchestra/testbench-dusk": "^8.0",
+    "phpunit/phpunit": "^9.0|^10.0",
```

Comment:
Might be best to migrate to PHPUnit 10 since both Laravel 10 and 11 supports it. PHPUnit 9 isn't supported in Laravel 11.